### ISSUE TITLE
Create newsletter

### ISSUE THREAD

**Issue by shahednasser** [shahednasser/awesome-resources#179]
I think creating a newsletter where people can sign up and receive weekly any new resources added would be helpful. However, I'm not sure the best way to do this since the resources are just added in the README.

Opening this for ideas if anyone has any.

**Comment by moingshaikh** [comment]
Hi @shahednasser,

I came to your repo from GitHub search page. 

I believe this can be achieved using @mailchimp. I am not very much sure about this but I have heard a few times that developers can use Mailchimp's embedded forms to offer a 'signup' feature on their GitHub pages and I am happy to try this if you see any benefit in this.

However, I'd also say that using Mailchimp will only create the form to collect subscriptions, it'll still require some external mechanism for you to be able to send them weekly updates/resources that are added here on GitHub.

**Comment by moingshaikh** [comment]
Also, do you need any help/contribution in adding new resources?

Thanks,
Moin

**Comment by diangogav** [comment]
perhaps we could take advantage of the webhooks
https://docs.github.com/en/rest/reference/repos#merge-a-branch

**Comment by shahednasser** [comment]
> Hi @shahednasser,
> 
> I came to your repo from GitHub search page.
> 
> I believe this can be achieved using @mailchimp. I am not very much sure about this but I have heard a few times that developers can use Mailchimp's embedded forms to offer a 'signup' feature on their GitHub pages and I am happy to try this if you see any benefit in this.
> 
> However, I'd also say that using Mailchimp will only create the form to collect subscriptions, it'll still require some external mechanism for you to be able to send them weekly updates/resources that are added here on GitHub.

Hi @moingshaikh thank you for your input. I don't think the problem is with the sign up form as I know of a lot of services that offer that. I'm just looking for a way to automate the newsletter based on the resources being added weekly.



> Also, do you need any help/contribution in adding new resources?
> 
> Thanks, Moin

Sure! Check the [contributing guidelines](https://github.com/shahednasser/awesome-resources/blob/6880e707cac65d27e7a86f8d853c0851264334ef/CONTRIBUTING.md) for qualified resources and how to contribute.

**Comment by shahednasser** [comment]
> perhaps we could take advantage of the webhooks
> https://docs.github.com/en/rest/reference/repos#merge-a-branch

That's a good idea @diangogav . I'll have to look more through it and see the best way to do this. If you have additional ideas on how we can benefit from it or you would like to work on this issue let me know.

**Comment by shahednasser** [comment]
Hello @srinivasthedeveloper the problem isn't with sending the email in particular, the problem is extracting the newly added resources weekly and making them the content of the newsletter. Since the resources are just added in the README it's harder to track the newer ones

**Comment by murtuzaalisurti** [comment]
maybe you could make a bot which collects some resources after you merge some pull requests and then send them to the the subscribers? just an idea

**Comment by shahednasser** [comment]
Yes @murtuzaalisurti that's the plan but I think we need to find an email service that we can post these resources to and this email service needs to send the newsletter weekly when there are new resources.

Let me know if you have any ideas regarding that.

**Comment by shahednasser** [comment]
Another idea I'm having is creating an RSS feed that is updated every time a PR is merged, which is probably the best direction to go in. That way an email service can get updates from the RSS feed and send them to subscribers

**Comment by murtuzaalisurti** [comment]
> Another idea I'm having is creating an RSS feed that is updated every time a PR is merged, which is probably the best direction to go in. That way an email service can get updates from the RSS feed and send them to subscribers

Yes, I think that's a better option.

**Comment by kshitijmishra27** [comment]
have did any thing regarding this newsletter etc ??? just curious to know about it

**Comment by shahednasser** [comment]
Hello @kshitijmishra27 no unfortunately. I haven't had the time to work on it. If anyone would like to work on it let me know!

**Comment by BhumikaJena** [comment]
I am interested in contributing. Is this still open to work on?

**Comment by heysagnik** [comment]
Please assign me

**Comment by iammuhammedfuhad** [comment]
Hi Iam intrested in cotributing to it , can you please assign me..?
